datacite-update-doi-handler/src/main/java/no/unit/nva/datacite/handlers/UpdateDoiEventHandler.java

```diff
@@ -125,6 +126,19 @@ private void handleDoi(DoiStateDto doiStateDto, DoiUpdateRequestEvent input, Doi
 
     }
 
+    private void handleRegisteredDoi(DoiUpdateRequestEvent requestEvent, Doi doi) {
+        logger.info(DOI_ALREADY_REGISTERED_MESSAGE, doi, requestEvent.getCustomerId(), requestEvent.getPublicationId());
+    }
+
+    private void handleDraftDoi(DoiUpdateRequestEvent updateRequestEvent, Doi doi) {
```

Comment:
would be better to name it deleteDraftDoi or something to better say what it does?